Example/Tests/GINICameraViewControllerTests.swift

```diff
@@ -69,5 +76,81 @@ final class CameraViewControllerTests: XCTestCase {
                       "multipageReviewBackgroundView should not be hidden after capture the second picture")
         
     }
+    
+    func testPickerCompletionBlockWhenNoErrors() {
```

Comment:
Either `WithNoErrors` or `WhenNoErrorsOccurred`. Otherwise it's not a correct sentence. Also applies for other tests in this class.